addons/website/static/src/js/editor/wysiwyg_multizone.js

```diff
@@ -68,6 +69,13 @@ DropzonePlugin.include({
     },
 });
 
+LinkPlugin.LinkPlugin.include({
+    _onDblclick: function (ev) {
+        if (!$(ev.target.closest('li')).find('.dropdown_mega_menu').length) {
+            this.show();
```

Comment:
You are extending a method without calling the parent implementation (_super).
If I understand what you are doing here, should not it be a simpler condition in the web_editor directly: "do not show if this is a dropdown-toggle" ?

Reply:
I initially put my condition in web_editor directly but you told me not to do it, no ? Yes, what you tell me seems also simpler to me

Reply:
I was probably telling you that you should not reference **mega menus** in web_editor